dashboard/src/reducers/namespace.ts

```diff
@@ -26,11 +26,24 @@ const namespaceReducer = (
   action: NamespaceAction | LocationChangeAction | AuthAction,
 ): INamespaceState => {
   switch (action.type) {
+    case getType(actions.namespace.receiveNamespace):
+      if (!state.namespaces.includes(action.payload.metadata.name)) {
+        return {
+          ...state,
+          namespaces: state.namespaces.concat(action.payload.metadata.name),
```

Comment:
Not sure whether the k8s API returns the namespace list ordered, but we probably want to ensure the same, so that after creating a namespace, the order of the list is the same?